### PR TITLE
Add ribosomal genes to qc

### PR DESCRIPTION
**Pull request by gwaybio**
We likely will want to further refine filtering to include all essential genes, but this is ok for now

### CODE REVIEW

File: 0.data-download/scripts/nbconverted/construct_gene_dictionary.py
Comment (Ecohen1763) [inline 0.data-download/scripts/nbconverted/construct_gene_dictionary.py:1]: Update documentation to reflect addition of new column.

**Review by Ecohen1763 — APPROVED** [review]
Looks all good, just one small comment. Other than that, I think it's good to merge 💪

File: 0.data-download/scripts/nbconverted/construct_gene_dictionary.py
Comment (gwaybio) [reply 0.data-download/scripts/nbconverted/construct_gene_dictionary.py:1]: great catch! Thanks!

added in 56ca5788ea4ef26e12662c061c0b0c1c06e10deb

**Comment by gwaybio** [comment]
Thanks @Ecohen1763 - I will now merge!

**Pull request merged**